src/main/java/seedu/address/model/person/Person.java

```diff
@@ -106,17 +122,16 @@ public boolean equals(Object other) {
         }
 
         // instanceof handles nulls
-        if (!(other instanceof Person)) {
+        if (!(other instanceof Person otherPerson)) {
             return false;
         }
-
-        Person otherPerson = (Person) other;
         return name.equals(otherPerson.name)
                 && phone.equals(otherPerson.phone)
                 && email.equals(otherPerson.email)
                 && address.equals(otherPerson.address)
                 && tags.equals(otherPerson.tags)
-                && remark.equals(otherPerson.remark);
+                && remark.equals(otherPerson.remark)
```

Comment:
remember to add birthday here (putting so merge conflict get removed faster)

Reply:
You do realise putting birthday here without the implementation of birthdya will not let the code compile